entity-framework/core/get-started/netcore/new-db-sqlite.md

```diff
@@ -1,100 +1,91 @@
+Tip: In a real application you would put each class in a separate file and put the connection string in a configuration file. To keep the tutorial simple, we are putting everything in one file.
 
-## Create your database
+## Create the database
 
-Now that you have a model, you can use migrations to create a database for you.
+Once you have a model, you can use migrations to create a database.
 
-* Run `dotnet ef migrations add MyFirstMigration` to scaffold a migration to create the initial set of tables for your model.
-* Run `dotnet ef database update` to apply the new migration to the database. Because your database doesn't exist yet, it will be created for you before the migration is applied.
+* Run `dotnet ef migrations add MyFirstMigration` to scaffold a migration and create the initial set of tables for the model.
+* Run `dotnet ef database update` to apply the new migration to the database. This command creates the database before applying migrations. 
 
-> [!NOTE]
-> When using relative paths with SQLite, the path will be relative to the application's main assembly. In this sample, the main binary is `bin/Debug/netcoreapp1.1/ConsoleApp.SQLite.dll`, so the SQLite database will be in `bin/Debug/netcoreapp1.1/blogging.db`.
+Note: When using relative paths with SQLite, the path will be relative to the application's main assembly. In this sample, the main binary is `bin/Debug/netcoreapp1.1/ConsoleApp.SQLite.dll`, so the SQLite database will be in `bin/Debug/netcoreapp1.1/blogging.db`.
 
-> [!TIP]
-> If you make future changes to your model, you can use the `dotnet ef migrations add` command to scaffold a new migration to make the corresponding schema changes to the database. Once you have checked the scaffolded code (and made any required changes), you can use the `dotnet ef database update` command to apply the changes to the database.
+Tip: If you make changes to your model, you can use the `dotnet ef migrations add` command to scaffold a new migration to make the corresponding schema changes to the database. Once you have checked the scaffolded code (and made any required changes), you can use the `dotnet ef database update` command to apply the changes to the database.
 >
 >EF uses a `__EFMigrationsHistory` table in the database to keep track of which migrations have already been applied to the database.
 
-> [!WARNING]
-> Migrations on SQLite do not support all schema changes due to limitations in SQLite itself. See [SQLite Limitations](../../providers/sqlite/limitations.md)
+Note: SQLite does not support all migrations (schema changes) due to limitations in SQLite. See [SQLite Limitations](../../providers/sqlite/limitations.md). For new development, consider dropping the database and creating a new one rather than using migrations when your model changes.
```

Comment:
This too I question the need for "note:".  The discussion of how to handle schema changes might fit better as a section at the end of the tutorial, after you've successfully created and accessed your db.